pkg/apis/types/types.go

```diff
@@ -104,3 +104,13 @@ const (
 	//TotalWaitTime is the max time duration to wait for doing snapshot operation on all the replicas
 	TotalWaitTime = 60
 )
+
+const (
+	// OpenEBSDisableDependantsReconcileKey is the annotation key that decides to create
+	// children objects with OpenEBSDisableReconcileKey as true or false
+	OpenEBSDisableDependantsReconcileKey = "reconcile.openebs.io/disable-dependants"
+
+	// ExistingPoolName is the name of the pool already present on the disk
+	// that needs to be imported and renamed
+	ExistingPoolName = "import.cspi.cstor.openebs.io/existing-pool-name"
```

Comment:
Let us make the `ExistingPoolName` to `OpenEBSCStorExistingPoolName` -- as this is under pkg/apis/types, it will cause conflicts if another engine wants to define a similar variable.

Reply:
Kiran,  we already specifying in `import.cspi.cstor.openebs.io` other than this should we need to explicitly specify it?

Reply:
@mittachaitu Here we are talking about the variable name instead of the key. As the import path will be `github.com/openebs/api/pkg/apis/types` the use of `types.ExistingPoolName` will be confusing to use in the code when some other engine also has a similar key.

Reply:
Ahh my bad though of the other way.